wigwam/_utils.py

```diff
@@ -203,6 +203,24 @@ def test_image(image: Image, expression: str) -> bool:
             raise
 
 
+def get_libdir(
+    base_tag: str,
+) -> str:
+    """Determine if the given image uses `lib64` or `lib` as its `LIBDIR`."""
+    # Unlike with the CMake Install step, `libdir` can be checked directly on this image
+    # because it has something at $INSTALL_PREFIX. Check the base tag for lib64 or lib.
```

Comment:
This comment doesn't seem to be applicable here. The image depends on the `base_tag` that the user provided -- it doesn't necessarily have something in `$INSTALL_PREFIX`.

Maybe we should add a note in the docstring that clarifies that we're assuming that the image contains a directory named `$INSTALL_PREFIX/lib` or `$INSTALL_PREFIX/lib64`.